docker/docker.mk

```diff
@@ -164,7 +164,7 @@ endif
 docker-push: SHELL := $(shell which bash)
 docker-push: docker-build
 	$(info Pushing $(DOCKER_IMAGE):$(DAPR_TAG) docker images ...)
-ifeq ($(TARGET_ARCH),amd64)
+ifeq ($(TARGET_ARCH),$(TARGET_ARCH_LOCAL))
```

Comment:
added the push change for consistency, but can revert if we want it hardcoded to amd64. I don't use the push in my case